### PR TITLE
Partial integration to the Browser Panel

### LINKED ISSUE #2101: [FEATURE][processing] Allow drag and drop of model files onto QGIS window
to execute them
Original commit: https://github.com/qgis/QGIS/commit/6300464275914ab8bfef6dd5eb2617bc9c9ff26e by nyalldawson

Unfortunately this naughty coder did not write a description... :-(

### PR DESCRIPTION
**Pull request by ghtmtt**
Goal: integrates with a brief description what is already well documented in the Browser Panel. The topic is Drag & Drop scripts (python especially). No images added just a few lines of more description

Ticket(s): fixes #2101 

- [x] Backport to LTR documentation is required

### CODE REVIEW

File: docs/user_manual/introduction/browser.rst [hunk 1]
```diff
@@ -15,10 +15,11 @@ browser.
...
 In addition, the Browser panel supports drag and drop of many QGIS
-resources, such as project files, processing scripts and processing
-models.
-Processing scripts and models can also be opened for editing in an
-external editor and the graphical modeller respectively.
+resources, such as project files, python scripts, processing scripts and 
+processing models.
+
+Python scripts, Processing scripts and Processing Models can also be opened for 
+editing in an external editor and the graphical modeller respectively.
```
Comment (DelazJ) [inline docs/user_manual/introduction/browser.rst:22]: We have three resources but two editors so I'm not sure "respectively" is correct to use here.

**Comment by ghtmtt** [comment]
> @ghtmtt I can't see the relation between your changes and the issue you refer to; that one is only about model files afaics (already documented from what I can see btw).

mmm why not? Isn't the issue describing that the user can D&D also models, besides of scripts py and processing scripts?

File: docs/user_manual/introduction/browser.rst (hunk 1 above)
Comment (ghtmtt) [reply docs/user_manual/introduction/browser.rst]: right, not sure either. Removed.

**Comment by ghtmtt** [comment]
@DelazJ should I rebase this and merge?

**Comment by DelazJ** [comment]
Yes, please do.

**Pull request merged**